src/Interpreters/ClusterProxy/SelectStreamFactory.cpp

```diff
@@ -169,7 +169,7 @@ void SelectStreamFactory::createForShardImpl(
             query_ast, header, context, processed_stage, shard_info.shard_num, shard_count, has_missing_objects));
     };
 
-    auto emplace_remote_stream = [&](bool lazy = false, time_t local_delay = 0)
+    auto emplace_remote_stream = [&](bool lazy = false)
```

Comment:
While you are here, can also add a comment for what `lazy` means

Reply:
done